src/megatron/bridge/training/optim.py

```diff
@@ -38,19 +34,13 @@ def setup_optimizer(
         scheduler_config: Configuration for the scheduler
         model: The model to optimize
         use_gloo_process_groups: Whether to use Gloo process groups
-        no_weight_decay_cond: Condition for parameters to exclude from weight decay
-        scale_lr_cond: Condition for parameters to scale learning rate
-        lr_mult: Learning rate multiplier
 
     Returns:
         tuple containing the optimizer and scheduler
     """
     optimizer = get_megatron_optimizer(
         optimizer_config,
         model,
```

Comment:
plz wire the config_override as well.
we can temporarily put "decoupled_min_lr" in to train config. and use the same logic in `setup` to do the same logic as mlm. https://github.com/NVIDIA/Megatron-LM/commit/7d3f4a012267465a919e48d10cc6b8bf6c784632#diff-7fc2a91343e49be3c1705b8237bbe437029132d8826e5c963e1b0b84d457bc38R1105-R1113